Move from third robot monitor

AriYu opened the pull request:
`third_robot_monitor`から`remote_monitor`として持ってきました。
このpkgがないと`waypoint_manager`の`catkin_make`が通りません。

AriYu committed:
[add] third_robot_monitorを実装

5m進んだらサービスをコールしてオドメトリを送信する。

AriYu committed:
[modify] パッケージ情報を変更

AriYu committed:
[modify] third_robot_monitor_serverをクラス化

AriYu committed:
[modify] 受け取った位置をmonitor_odomとしてpublishするように変更

AriYu committed:
[moidfy] rvizで見れるようにtfを追加

AriYu committed:
[fix] 無駄にpublishしないように

MoriKen254 committed:
[add] 地図画像読込機能実装

AriYu committed:
[modify] odomからamclをサブスクライブする用に変更

MoriKen254 committed:
[modify] パラメータで画像ファイル調整できるよう修正

MoriKen254 committed:
Merge branch 'third_robot_monitor' of https://github.com/Nishida-Lab/TC2015 into third_robot_monitor

MoriKen254 committed:
[modify] コマンド追加

MoriKen254 committed:
[add] README.md を追加。

MoriKen254 committed:
[modify] 記述修正。

MoriKen254 committed:
[modify] launch修正

MoriKen254 committed:
Merge branch 'third_robot_monitor' of https://github.com/Nishida-Lab/TC2015 into third_robot_monitor

MoriKen254 committed:
[modify] 記述を修正。

MoriKen254 committed:
Update README.md

MoriKen254 committed:
Update README.md

MoriKen254 committed:
Update README.md

RyodoTanaka committed:
[fix] fix compile error

1. cv::cos cv::sin でエラーが出ていたのを修正。
2. CMakeLists.txt にOpenCVのコンパイルオプションがついていなかったのを修
正。

MoriKen254 committed:
[fix] fixed a composition error

MoriKen254 committed:
[modify] monitor関係一気に修正

MoriKen254 committed:
[fix] typo.

MoriKen254 committed:
[add] 仕様変更に合わせて記述を追加．

MoriKen254 committed:
[add] デモ用画像追加

MoriKen254 committed:
[add] デモ動画を追加．

MoriKen254 committed:
[add] monitorについてGazeboでの起動方法を追加．

AriYu committed:
add vpn setting script

MoriKen254 committed:
[add]vpn server用設定スクリプト追加

MoriKen254 committed:
Merge branch 'master' of github.com:CIR-KIT/third_robot_pkg

AriYu committed:
modify ファイルを移動した

AriYu committed:
Merge branch 'master' of github.com:CIR-KIT/third_robot_pkg

MoriKen254 committed:
[fix] monitor用スクリプト保存場所修正

MoriKen254 committed:
[add] monitorで人の位置を表示する機能を追加．

MoriKen254 committed:
[modify] monitorのスクリプトの権限付与

MoriKen254 committed:
[fix] デバッグ用の記述をコメントアウト

MoriKen254 committed:
[modify] ロボット位置を矢印先端の形状に変更．

MoriKen254 committed:
[add] monitorの見た目改善．

MoriKen254 committed:
[modify] README for monitor

AriYu committed:
modified make disable the 3d urg for a performance

AriYu committed:
fix include file name for modified pkg name

AriYu committed:
fix package name

AriYu committed:
fix project and executable node name

AriYu committed:
Merge branch 'master' of github.com:CIR-KIT/remote_monitor

Conflicts:
	README.md

MoriKen254 committed:
[modify] modify dependnecy of monitor srv

MoriKen254 commented:
https://github.com/CIR-KIT/waypoint_manager/pull/1

での依存関係修正に伴い、こちらのメッセージ生成を無効にしました。

forno committed:
Delete comment of for create package

forno committed:
Migrate format 1 to 2

forno committed:
Fix dependency for all source code

forno committed:
Order include by alfabet and divide by where

stdio.h はシステムから探してくるので改行してみやすく
rosシステムはCMakeに/opt/ros/<distribution>を自動で追加するけど
通常パスが通っているところでは無いため""で参照するのがいいかと思います。

それはともかく<>と""のどちらかに統一すべきだと思ったので、
"ros/ros.h"に合わせてcatkinから探すものは全て""にしました。

forno committed:
Fix dependency of unnessary depend

この行は完全に要らないし、むしろコンパイルの邪魔

forno committed:
Fix dependency of system library that is Boost

AriYu committed:
add add_dependencies to build srv file befor remote_monitor pkg

AriYu committed:
Merge pull request #2 from CIR-KIT/fix-dependency-for-all-code

Fix dependency for all code
